app/assets/javascripts/snowcrash/modules/issues/evidence.js.coffee

```diff
@@ -4,4 +4,9 @@ jQuery ->
     $('#evidence-tabs .tab-pane').first().addClass('active')
 
     $('#evidence-tabs a[data-toggle="tab"]').on 'shown', (ev)->
-      $('#evidence-tabs #evidence-host-list').css('height', $('#evidence-tabs .tab-content').css('height'))
+      tabContentHeight = $('#evidence-tabs .tab-content').css('height')
+      $tabs            = $('#evidence-tabs #evidence-host-list')
+      if $tabs.height() < tabContentHeight
```

Comment:
this isn't working as tabContentHeight comes with `px` (e.g. `80px`) and `.height()` is just a number.

But even fixing that it doesn't work because L12 doesn't "reset" the value, it seems to act just as a no-op leaving it with whatever CSS height it had before the call.